argocd-helmfile/Dockerfile

```diff
@@ -1,37 +1,32 @@
-ARG ARGO_VERSION=v0.12.3
+FROM argoproj/argocd:v1.6.1
 
-FROM argoproj/argocd:${ARGO_VERSION}
-
-ARG HELM_DIFF_VERSION=master
-ARG HELM_SECRETS_VERSION=master
-ARG HELM_IMPORT_VERSION=0.2.1
-ARG HELM_TILLER_VERSION=v0.6.7
-ARG HELMFILE_VERSION=0.45.3
-
-LABEL version="0.12.3"
-LABEL maintainer="ozaki@chatwork.com"
+LABEL version="v1.6.1"
```

Comment:
@taishin 
This version number is the tag name for `argocd-helmfile`.
The dockerfiles repository uses `X.Y.Z` format tags, so the special Please use this format unless you have a good reason to do so.

Reply:
@taishin 
The mods to be updated are argocd and helmfile.
Therefore, when the helmfile is updated, this version of the specification will overwrite the same tag.